Source/RapyutaSimulationPlugins/Public/Core/RRNetworkGameState.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file RRNetworkGameState.h
+ * @todo add doc
+ * @copyright Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
+ */
+#pragma once
+
+// UE
+#include "CoreMinimal.h"
+#include "GameFramework/GameState.h"
+
+#include "RRNetworkGameState.generated.h"
+
+UCLASS()
+class RAPYUTASIMULATIONPLUGINS_API ARRNetworkGameState : public AGameState
+{
+    GENERATED_BODY()
+
+public:
+    ARRNetworkGameState();
+
```

Comment:
Do we need a new class just for this method?

Reply:
probably can have another way, but I saw people use gamestate for stuff like this so I followed

Reply:
also allowed for overriding the server time functions